src/main/resources/static/headline.html

```diff
@@ -0,0 +1,5 @@
+<nav>
```

Comment:
why is there a navtag around this?

Reply:
I thought id would be needed to implement

Reply:
nope. https://developer.mozilla.org/en-US/docs/Web/HTML/Element/nav?retiredLocale=de